src/server/JasmineGraphServer.cpp

```diff
@@ -471,9 +470,33 @@ void JasmineGraphServer::resolveOperationalGraphs(){
 
 }
 
-//TODO:Needs to implement this method
 void JasmineGraphServer::deleteNonOperationalGraphFragment(int graphID) {
+    /**
```

Comment:
Move this multi-line comment above function signature. Follow the [guideline](https://developer.lsst.io/v/DM-5063/docs/cpp_docs.html#function-method-definitions).

Reply:
Followed the guidelines and moved the multi-line comment above the function signature. Used @param to before describing each input parameter. Removed "INPUT" and "OUTPUT" parts of the comment